code/datums/loadout/loadout_helper.dm

```diff
@@ -0,0 +1,55 @@
+///Return a new empty loayout
+/proc/create_empty_loadout(name = "Default", job = MARINE_LOADOUT)
+	var/datum/loadout/empty = new
+	empty.name = name
+	empty.job = job
+	empty.item_list = list()
+	return empty
+
+///Return true if the item was found in a linked vendor
+/proc/is_savable_in_loadout(saved_item_type)
+	for(var/type in GLOB.loadout_linked_vendor)
+		for(var/datum/vending_product/item_datum in GLOB.vending_records[type])
+			var/product_path = item_datum.product_path
+			if(product_path == saved_item_type)
+				return TRUE
+	return FALSE
+
+///Return true if the item was found in a linked vendor and successfully bought
+/proc/buy_item_in_vendor(item_to_buy_type)
+	for(var/type in GLOB.loadout_linked_vendor)
+		for(var/datum/vending_product/item_datum in GLOB.vending_records[type])
+			var/product_path = item_datum.product_path
+			if(product_path == item_to_buy_type && item_datum.amount != 0)
+				item_datum.amount--
+				return TRUE
+	return FALSE
+
+/// Will put back an item in a linked vendor
+/proc/sell_back_item_in_vendor(item_to_give_back_type)
+	for(var/type in GLOB.loadout_linked_vendor)
+		for(var/datum/vending_product/item_datum in GLOB.vending_records[type])
+			var/product_path = item_datum.product_path
+			if(product_path == item_to_give_back_type)
+				item_datum.amount++
+				return 
+
+///Return wich type of item_representation should representate any item_type
+/proc/item_representation_type(item_type)
```

Comment:
item2representation_type is the standard formatting for this and you should also consider putting it in the type2thing file

Reply:
looking at type2type.dm, i'm not really convinced. Think it's better here